cli/catalog-api-v1/README.md

```diff
@@ -34,9 +34,15 @@ Mid-term updating the client library and proposing it as a PR will be done by au
 [mitmproxy](https://mitmproxy.org/) can be used to debug requests and responses from the Catalog API.
 
 1. Start the interface and leave it running in a separate terminal:
-
-        mitmproxy
-
+    ```
+    nix run github:flox/nixpkgs/stable.20240203#mitmproxy
+    ```
+    Note that `mitmproxy` has been broken on Darwin for some time now,
+    hence the need to run it from an old nixpkgs revision.
+    Of course once we implement catalog binary cache verification
+    then using this will be as simple as `flox install mitmproxy`,
+    but in the meantime we just run it from the most recent stable
+    nixpkgs revision on which it is known to build.
```

Comment:
(non-blocking) We can also `flox install mitmproxy@10.1.6` to get the same working version today.